src/chef_authn.erl

```diff
@@ -161,14 +162,19 @@ process_key({'Certificate', _Der, _} = CertEntry) ->
     Cert = public_key:pem_entry_decode(CertEntry),
     TbsCert = Cert#'Certificate'.tbsCertificate,
     public_key_from_spki(TbsCert#'TBSCertificate'.subjectPublicKeyInfo);
-process_key({'PrivateKeyInfo', _, _} = Entry) ->
+process_key({'PrivateKeyInfo', Der, _} = Entry) ->
     KeyInfo = public_key:pem_entry_decode(Entry),
-    KeyAlgorithmInfo = KeyInfo#'PrivateKeyInfo'.privateKeyAlgorithm,
-    case KeyAlgorithmInfo of
-        #'PrivateKeyInfo_privateKeyAlgorithm'{algorithm=?'rsaEncryption'} ->
-            PrivateKey = KeyInfo#'PrivateKeyInfo'.privateKey,
+    io:format("KeyInfo ~p~n", [KeyInfo]),
```

Comment:
[nit] If we want to print from this library we should probaby have some configuration for a logging module and/or use lager directly.

Reply:
No that was a goof